### PR TITLE
Combines the major disease outbreak event with the moderate.

### PR DESCRIPTION
**Pull request by alex-gh**
## What Does This PR Do
Combines the major and moderate disease outbreak into one.

## Why It's Good For The Game
Major disease outbreak was firing too often.
Even major diseases are not as catastrophic for the station as other events in the major category thus "wasting" a midround with a non-event.

## Testing
Verified that the disease generation, ghost message and infection are still working as intended.

## Changelog
:cl: uc_guy
tweak: Combines the major and moderate disease outbreak events into one.
/:cl:

### CODE REVIEW

File: code/modules/events/disease_outbreak.dm [hunk 1]
```diff
@@ -1,3 +1,6 @@
+#define MAJOR_INTO_MODERATE TRUE  //If true folds die major diseases/symptoms into moderate ones
```
Comment (hal9000PR) [inline code/modules/events/disease_outbreak.dm:1]: the compiler doesnt like this. this makes always true statements. I would prefer to just not have the option to change this. we can always just revert it if its bad

**Review by hal9000PR — CHANGES_REQUESTED** [review]

File: code/modules/events/disease_outbreak.dm (hunk 1 above)
Comment (alex-gh) [reply code/modules/events/disease_outbreak.dm:1]: I put it into an #ifdef which seems like the proper way to do it,

I mean, I can get rid of it being changeable, I just think still having that logic there might be useful for the virology expansion/rework in the future.

Comment (hal9000PR) [reply code/modules/events/disease_outbreak.dm:1]: I would rather avoid using `ifdef`

Comment (alex-gh) [reply code/modules/events/disease_outbreak.dm:1]: Alright. Will rework it tonight.

Comment (alex-gh) [reply code/modules/events/disease_outbreak.dm]: This should be it then, unless I missed something.
Works in testing and the list in the debugger is correct.

File: code/modules/events/disease_outbreak.dm [hunk 2]
```diff
@@ -22,35 +21,27 @@ GLOBAL_LIST_EMPTY(current_pending_diseases)
 			if(EVENT_LEVEL_MUNDANE)
 				virus = pick(diseases_minor)
 			if(EVENT_LEVEL_MODERATE)
```
Comment (SteelSlayer) [inline code/modules/events/disease_outbreak.dm:23]: Should this be switched to `if(EVENT_LEVEL_MODERATE, EVENT_LEVEL_MAJOR)`? Right now it looks like if major is rolled, it makes a stack trace, then makes the virus a cold, which seems odd.

Comment (SteelSlayer) [reply code/modules/events/disease_outbreak.dm:23]: Ah, right, I see.

**Review by SteelSlayer — APPROVED** [review]

**Review by hal9000PR — APPROVED** [review]

**Pull request merged**